ui/desktop/tests/acceptance/projects/targets-test.js

```diff
@@ -209,6 +209,17 @@ module('Acceptance | projects | targets', function (hooks) {
     await visit(urls.targets);
   });
 
+  test('cannot navigate to a target without proper authorization', async function (assert) {
+    assert.expect(1);
+    later(async () => {
+      run.cancelTimers();
+      instances.target.authorized_actions =
+        instances.target.authorized_actions.filter((item) => item !== 'read');
+      assert.notOk(find('main tbody .rose-table-header-cell:nth-child(1) a'));
+    }, 750);
+    await visit(urls.target);
```

Comment:
Isn't this `urls.targets` so that targets list view is loaded and links in listed table can be verified?

Reply:
you are right. this should be `urls.targets` I will update it. thank you!